lib/bootstrap.js

```diff
@@ -0,0 +1,65 @@
+var spawn = require('child_process').spawn;
+var path = require('path');
+var sleep = require('sleep').sleep;
+
+function Bootstrap(codecept, callback) {
+
+  var phantomjs, seleniumPid, callbackCalled = false;
+
+  (function () {
+
+    var protractor = codecept.config.helpers.Protractor;
+    if (protractor && protractor.browser) {
+      if (/phantom/.test(protractor.browser)) {
```

Comment:
I'd suggest to move phantomjs initialization and webdriverio initialization into functions (will make the code more readable)